### ISSUE TITLE
slim retrain inception no validation loss

### ISSUE THREAD

**Issue by neouyghur** [tensorflow/models#1874]
I use slim API retrain the Inception network. However, I find the slim does not return the validation loss at all. Normally, we should have both train loss and validation loss.

```
INFO:tensorflow:global step 40640: loss = 1.0105 (1.39 sec/step)
INFO:tensorflow:global step 40650: loss = 1.0092 (1.39 sec/step)
INFO:tensorflow:global step 40660: loss = 1.0107 (1.40 sec/step)
```
Besides that, does slim support the tensorboard? I did not find any tutorial on it.

**Comment by korrawat** [comment]
Slim supports TensorBoard. The [walkthrough](https://github.com/tensorflow/models/blob/master/slim/slim_walkthrough.ipynb) mentions it, but that doesn't clearly say how to use it.

Maybe an example of how to use TensorBoard, which would be just a couple lines, can be added to the [README](https://github.com/tensorflow/models/blob/master/slim/README.md) (and the walkthrough)?

**Comment by neouyghur** [comment]
You are right, slim supports tensor board. I did not access the tensor
board, because there is no log file exist at the beginning.  Later, I
accessed the tensor board, but there is no graph at all.

On Fri, Jul 7, 2017 at 6:27 AM, James Pruegsanusak <notifications@github.com

**Comment by cy89** [comment]
@sguada can you please answer @neouyghur 's first question, about whether it's possible to get the validation loss from slim?

**Comment by neouyghur** [comment]
@cy89 I am using the the train_image_classifier.py for retraining the inception network. The reason I need the validation loss is that to overcome overfitting or underfitting. Without validation loss, I don't know when to stop training.

**Comment by wt-huang** [comment]
Closing as this is resolved.
